cmd/swarmd/main.go

```diff
@@ -55,7 +55,6 @@ var (
 			}
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			ctx := context.Background()
```

Comment:
@stevvooe I have no problem either way. It's good to have #1254 which will make things  a bit more clear about the context usage.